### PR TITLE
Update ja.json by SCSK

### PR DESCRIPTION
**Pull request by scsk-iot-tech**

### CODE REVIEW

File: i18n/ja.json [hunk 1]
```diff
@@ -506,7 +511,7 @@
   "COPY_TO_CLIPBOARD": "クリップボードにコピーしました。",
 
   "ARE_YOU_SURE_LEAVE_PROJECT": "プロジェクトから解除してもよろしいですか？",
-  "LEAVE": "解除",
+  "LEAVE": "OK",
```
Comment (Dyskovsky) [inline i18n/ja.json]: Note that from Convel 0.14.0 translation key "LEAVE"  is used in two places; to confirm leave Rule Wizard and as tooltip for the button to leave from a project. We will create a new key to allow multiple translations. Please confirm if you want to merge it to master.

Comment (scsk-iot-tech) [reply i18n/ja.json]: If the key will be created soon later, please discard changing for this "LEAVE" field.

Comment (Dyskovsky) [reply i18n/ja.json]: We created a new translation key "LEAVE_PROJECT" but it will be used in Convel from version 0.26.0

**Comment by scsk-iot-tech** [comment]
We close this pull request at this moment.
We'll make another pull request after translating missing field including "LEAVE_PROJECT".

**Pull request closed without merging**